test/test_charmstore_apiv4.js

```diff
@@ -166,6 +166,13 @@ describe('Charmstore API v4', function() {
       charmstore._lowerCaseKeys(uppercase, host);
       assert.deepEqual(host, { baz: '1', foo: { bar: { baz: '1'}}});
     });
+
+    it('can skip one level of keys in an object', function() {
+      var uppercase = { Baz: '1', Foo: { Bar: { Baz: '1' }}, Fee: '2'};
+      var host = {};
+      charmstore._lowerCaseKeys(uppercase, host, 0);
```

Comment:
I'm surprised by this usage. I expected 'exclude' to be a positive integer that counts down.